cmd/podman/pods/create.go

```diff
@@ -147,6 +148,12 @@ func create(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	fmt.Println(response.Id)
+	if len(podIDFile) > 0 {
+		if err = ioutil.WriteFile(podIDFile, []byte(response.Id), 0644); err != nil {
+			return errors.Wrapf(err, "failed to write pod ID to file %q", podIDFile)
+		}
+	} else {
```

Comment:
Should this be an else? I think we always print response.Id even if we have a pod ID file - that's how `--cidfile` works

Reply:
Fixed.